meta/meta.go

```diff
@@ -67,15 +67,14 @@ var (
 )
 
 var (
-
 	// ErrDBExists is the error for db exists.
-	ErrDBExists = terror.ClassMeta.New(codeDatabaseExists, "database already exists")
+	ErrDBExists = terror.ClassMeta.New(mysql.ErrDBCreateExists, mysql.MySQLErrName[mysql.ErrDBCreateExists])
 	// ErrDBNotExists is the error for db not exists.
-	ErrDBNotExists = terror.ClassMeta.New(codeDatabaseNotExists, "database doesn't exist")
+	ErrDBNotExists = terror.ClassMeta.New(mysql.ErrBadDB, mysql.MySQLErrName[mysql.ErrBadDB])
```

Comment:
the message of `mysql.MySQLErrName[mysql.ErrBadDB]` is `"Unknown database '%-.192s'"`, as you can see, it need paramatter, but we use `ErrDBNotExists ` won't have parameter.